src/App.vue

```diff
@@ -17,20 +17,21 @@
             </div>
             <div id="navbarMenuHeroA" class="navbar-menu">
               <div class="navbar-end">
-                <a class="navbar-item is-active">
+                <a class="navbar-item is-active" href="https://pangeo-forge.org">
                   Home
                 </a>
-                <a class="navbar-item">
+                <a class="navbar-item" href="https://pangeo-forge.readthedocs.io/en/latest/recipe_box.html">
```

Comment:
Ideally this would point to an actual list of recipes. The "Recipe Box" page is pretty bare right now. I guess we can always update later as we fill things in.

Reply:
This would be an HTML-rendered list from a database of recipes, in which recipes would be registered at build-time? Perhaps then, this will not be a list, but a browsable catalog of its own? Thinking out loud here, but wondering if this is something that could be handled with STAC / STAC Browser as well...

Reply:
I think it would be something that would be populated by some API call. But maybe not STAC. STAC will catalog Datasets, not Recipes.

One idea would be to use the [GitHub API](https://docs.github.com/en/rest) to get a list of all the feedstock repos in the pangeo-forge org.